index.md

```diff
@@ -195,6 +196,14 @@ Operations:
 
 This is all data specific to a care team - its messaging, and questionnaires related to patient care.
 
+### Appointments
+
+The post to appointments allows user to create a calendar event.
```

Comment:
Probably can drop this single line description as seem redundant to the operation's copy.

Reply:
agree